packages/adapters/database/db/migrations/20230213141356_root-message-sent-status.sql

```diff
@@ -0,0 +1,9 @@
+-- migrate:up
+ALTER TABLE root_messages
+ADD COLUMN sent_time timestamp with time zone,
```

Comment:
Minor:
We have been storing timestamps as integers. This deviates from it.

Reply:
Yep fixed this.